### ISSUE TITLE
The Oxford Common File Layout

### ISSUE THREAD

**Issue by jwikle** [imls-dmt/resources-workflow#219]
Information about registered data management training resource

- [ ] Date resource added: 04/16/2019
- [ ] Registered by: jwikle
- [ ] Clearinghouse URL: http://dmtclearinghouse.esipfed.org/node/10386

Tracking: @karlbenedict @njhoebel

**Comment by raisaionin** [comment]
This resource no longer exists; the URL is invalid; 404 Not Found Error.
https://zenodo.org/record/2642296

Remove from DTMC.

**Comment by njhoebel** [comment]
When I checked this in June, the resource was available, so it's possible that Zenodo was down when Raisa checked this resource.  I recall that happening before, and around that time.  So, I have removed the "Remove" label and changed the following:
1.  Added more descriptive information about the context of this presentation based on information from the first & 2nd slides.
2.  Changed the authoring organization to be that of the presenter (University of Oxford, Bodleian Libraries.
3.  Changed the publisher to RDA after adding a new organization for the Research Data Alliance (an overall organization under which RDA/USA is organized).
4.  Added a keyword for "data preservation".
5.  Added a learning resource type of "lesson".
6,  Added the DOI information.
7. Moved to Publish.
8.  Raisa to review again and add RI_reviewed label again, then move to Done.

**Comment by raisaionin** [comment]
Thanks, Nancy!All done.
Zenodo is back and functioning.


Raisa.    On Saturday, June 22, 2019, 7:12:30 PM EDT, Nancy J. Hoebelheinrich <notifications@github.com> wrote:  
 
 
When I checked this in June, the resource was available, so it's possible that Zenodo was down when Raisa checked this resource. I recall that happening before, and around that time. So, I have removed the "Remove" label and changed the following:
   
   - Added more descriptive information about the context of this presentation based on information from the first & 2nd slides.
   - Changed the authoring organization to be that of the presenter (University of Oxford, Bodleian Libraries.
   - Changed the publisher to RDA after adding a new organization for the Research Data Alliance (an overall organization under which RDA/USA is organized).
   - Added a keyword for "data preservation".
   - Added a learning resource type of "lesson".   
6, Added the DOI information.
   - Moved to Publish.
   - Raisa to review again and add RI_reviewed label again, then move to Done.
